wacksbywarby/tests/test_wack.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+
+from wacksbywarby.wack import main
+from wacksbywarby.db import Wackabase
+
+
+@pytest.mark.parametrize(
+    "changes, num_additional_sales, num_expected_discord_messages",
+    [
+        # None for num_additional_sales means we'll calculate a proper number
+        ([], None, 0),
+        ([("957749348", -1)], None, 1),
```

Comment:
can you comment with what case these are as well? aka 1 sale, increase in inventory (not handled yet so no messages expected)